EnglishFilter/sections/specific.txt

```diff
@@ -3002,6 +3002,7 @@ allcelebs.club##a[href^="http://www.searchcelebrityhd.com/"]
 ettv.to##.vp_ab
 ettv.to##.imdb > div[class^="download_liinks"]
 looper.com##.recommended-heading
+xxxdessert.com##.fp-ui > div[style^="position: absolute; inset: 0px;"]
```

Comment:
Not working on my end. Search for `html.php?aid=pause_html` in the filters and make similar rule.